mapie/mondrian.py

```diff
@@ -0,0 +1,438 @@
+from copy import deepcopy
+from typing import Iterable, Optional, Tuple, Union, cast
+
+import numpy as np
+from sklearn.utils.validation import _check_y, check_is_fitted, indexable
+
+from mapie.calibration import MapieCalibrator
+from mapie.classification import MapieClassifier
+from mapie.conformity_scores import (
+    AbsoluteConformityScore,
+    APSConformityScore,
+    GammaConformityScore,
+    LACConformityScore,
+    NaiveConformityScore,
+    TopKConformityScore
+)
+from mapie.multi_label_classification import MapieMultiLabelClassifier
+from mapie.regression import (
+    MapieQuantileRegressor,
+    MapieRegressor,
+    MapieTimeSeriesRegressor
+)
+from mapie.utils import check_alpha
+from mapie._typing import ArrayLike, NDArray
+
+
+class Mondrian:
+    """Mondrian is a method that allows to make  perform conformal predictions
```

Comment:
Detail constraints here or in public methods docstring rather than in private methods docstrings.

Reply:
done